experiments/pipeline/README.md

```diff
@@ -0,0 +1,8 @@
+Data flow example that shows reading from a kafka queue and writing to HBase.
```

Comment:
Since we requires specific format of the message. 
In order to run this example, we also need to checkin the data generation part.